app/views/share/_post.html.erb

```diff
@@ -0,0 +1,20 @@
+<ul class= "list-unstyled align-items-center d-flex flex-column align-items-center w-75">
+    <li class="card mb-3 w-75 mt-4" >
+      <div class="col-md-12">
+      <div class="card-body">
+      <h4 class="card-title">Post #<%= post.id %></h4><br>
+      <h3 class="card-title"><%= post.title %></h3>
+      <p class="card-text" ><%= post.text %></p>
+       <p class="card-text text-end">
+        <small class="text-muted">
+          comments: <%= !post.comments_counter ? 0 :  post.comments_counter %>,
+          likes: <%= !post.likes_counter ? 0 :  post.likes_counter %>
```

Comment:
- [ ] I was unable to create a `like` in the console in order to check the views.  Tried several times:

![image](https://user-images.githubusercontent.com/57421823/166866380-a3bc2c02-8620-416f-ae58-85862c9b14fe.png)

Please, double-check `likes` can be created 💪